src/execution/execute.ts

```diff
@@ -152,6 +154,13 @@ export interface ExecutionArgs {
   fieldResolver?: Maybe<GraphQLFieldResolver<any, any>>;
   typeResolver?: Maybe<GraphQLTypeResolver<any, any>>;
   subscribeFieldResolver?: Maybe<GraphQLFieldResolver<any, any>>;
+  /**
+   * Set to `false` to disable error propagation. Experimental.
+   * TODO: describe what this does
```

Comment:
Outstanding TODO. We might want to say something here about how this is being used and include a link to the RFC or spec edits.